build.gradle.kts

```diff
@@ -38,67 +37,66 @@ plugins {
     id("org.asciidoctor.jvm.convert") version "3.3.2"
 
     // documentation
-    id("org.jetbrains.dokka") version "1.7.10"
-
-    // code analysis for kotlin
-    id("io.gitlab.arturbosch.detekt") version "1.21.0"
+    id("org.jetbrains.dokka") version "1.9.10"
 
     // plugin for publishing to Gradle Portal
-
-    id("com.gradle.plugin-publish") version "1.2.0"
+    id("com.gradle.plugin-publish") version "1.2.1"
 }
 
 // release configuration
 group = "com.intershop.gradle.version"
 description = "Gradle SCM version plugin - SCM based version handling for Gradle"
-version = "1.10.0"
+// apply gradle property 'projectVersion' to project.version, default to 'LOCAL'
+val projectVersion : String? by project
+version = projectVersion ?: "LOCAL"
 
-
-val sonatypeUsername: String by project
+val sonatypeUsername: String? by project
 val sonatypePassword: String? by project
 
 repositories {
     mavenCentral()
+    mavenLocal()
 }
 
+val pluginUrl = "https://github.com/IntershopCommunicationsAG/${project.name}"
 gradlePlugin {
+    website = pluginUrl
+    vcsUrl = pluginUrl
     plugins {
-        create("gitflowVersionPlugin") {
+        create(project.name) {
```

Comment:
This will change the name to `gitflow-version-gradle-plugin` (I think, according to the `settings.gradle.kts`). Is this just a display change, or will that break things?

Reply:
That is just an internal name.